package.json

```diff
@@ -5,7 +5,7 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "webpack --mode production && yarn install --modules-folder ./build/node_modules --production --frozen-lockfile",
+    "build": "ENVIRONMENT=production webpack --mode production && yarn install --modules-folder ./build/node_modules --production --frozen-lockfile",
```

Comment:
We don't really need the `--production` flag to `webpack` here if we're passing the env variable.